weaver/wps_restapi/swagger_definitions.py

```diff
@@ -228,6 +233,23 @@ class URL(ExtendedSchemaNode):
     format = "url"
 
 
+class DateTimeInterval(ExtendedSchemaNode):
+    schema_type = String
+    description = "DateTime format against ogcapi, \
+                    to get values before a certain date-time use '../' before the date-time \
+                    to get values after a certain date-time use '/..' after the date-time like the example \
+                    to get values between two date-times use '/' between the date-times \
+                    to get values with a specific date-time just pass the datetime"
```

Comment:
format: 
```
description = (
    "..."
    "..."
)
```

Be careful of extra spaces or newlines because the concatenated strings preserve them as is, which makes the OpenAPI format incorrect. 

You can try to call `print(description)` to see how it looks like to validate. 

Please replace *ogcapi* by *OGC-API - Processes*.
Make sure the punctuation of the sentences is correct.